### DISCUSSION TITLE
Create and Import Map from Roadrunner

### DISCUSSION THREAD

**Discussion by Dr2546** [carla-simulator/carla#7956] [Q&A] [answered]
Hello all,

I want to create and import Map from Raodrunner , My specs are
OS : Windows 11
CARLA : 0.9.15 download zip file from https://github.com/carla-simulator/carla/releases/tag/0.9.15
Roadrunner : R2023a

Is it possible? I read docs it say that CARLA should be built from source or OS is Ubuntu which neither meet my conditions.

Also this may off the topic but in my carla root folder I can't find ```Import``` directory, but CARLA works fine so maybe because I download zip not built from source? 

Thanks

**Comment by 15259400687** [comment]
if you wanna import your custom map，you should in CARLA built from source.Given the docs of Carla,if your carla's version is download from zip,you must do it in docker.

**Reply by Dr2546** [reply]
Hi, thanks for an answer.

So I try to export "scene" from Roadrunner and get xodr file. and then import this file to Roadrunner with a sample car with this [tutorial ](https://carla.readthedocs.io/en/latest/adv_opendrive/) and everything seems to work ok but there are few mistakes and questions.

1. Does scene and map are same thing in CARLA? document say that CARLA should built from source or atleast use Docker but I somehow use only xodr file and get result

2. I can control my car with ```carla.VehicleControl``` but when I use ```carla.VehicleAckermannControl``` car seems stutter and can't move.
Is this because of my method? because if I use other map like ```Town10``` it work fine

Thanks in advances

**Reply by 15259400687** [reply, marked as answer]
I'm sorry for replying you so late. For the first question, from my point of view, the concepts of scene and map are not the same. The scene also includes other materials that are not related to traffic, such as buildings and vegetation. The map is the most basic part for the vehicle to run. The reason why you can run the xodr file in the precompiled version is that you don't need to modify the assets in it. Just running the xodr file will make the world appear very empty, without other buildings or vegetation, etc. I don't know much about the second question. I used manual_control.py in the examples provided by Carla to test the map.

**Reply by Dr2546** [reply]
Thank you so much for your reply, I get it now about scene and map and it seems like what I really want is just a empty map with some road. and my second question is quite similar to this [issue](https://www.github.com/carla-simulator/carla/issues/6953) so maybe this is not because a map is create by Roadrunner.
